raw/style.css

```diff
@@ -26,15 +26,23 @@ a {
   text-decoration: none;
   color: #ffffff;
 }
-a:focus,
-button:focus {
+a:focus {
   outline: none;
   border: 2px solid #fb37f1;
+  border-radius: 15px;
   box-shadow: 0px 0px 4px #fb37f1;
+  padding: 0px 10px;
 }
 button {
   cursor: pointer;
 }
+
+button:focus {
+  outline: none;
```

Comment:
I've got a (probably annoying) habit of asking for css attributes to be alphabetical. Avoids accidental repetition and makes code review easier when commits ebb and flow over existing content.

Not something you need to do now but not a bad habit to get into for future (in my opinion).

Reply:
This would become a ballache and just slow down development time. It's only worth enforcing such rules through automation tools that run on e.g. a commit hook.

A block of CSS shouldn't be so long you can't spot duplication; if it is, it's probably not broken up enough (for example, following OOCSS principles.) I advise having a linter for detecting redundant/duplicated CSS.